inventory/host_vars/openfoodnetwork.net/config.yml

```diff
@@ -1,4 +1,20 @@
 ---
+
 domain: openfoodnetwork.net
```

Comment:
hey @mkllnk from a link you sent me some days ago I thought we should always have www in our domains. Any reason for not using in?

Reply:
That wasn't my decision. I'm not sure who set it up initially, but I guess we just kept it the same as it was. Don't change a running system.

While I understand the points for having a non-apex domain, I never encountered a real issue with OFN servers because of this. We are not using aliases, we are not using Cloudflare or similar providers, we just have one server with a static IP address.

Reply:
would you mind sharing it again :trollface: ?

Reply:
@sauloperez do you mean the link? I think it was this one: https://www.yes-www.org/why-use-www/